tests/react-native/End2End/package.json

```diff
@@ -7,9 +7,9 @@
     "test": "jest"
   },
   "dependencies": {
-    "@aws-sdk/client-cognito-identity": "1.0.0-ci.0",
-    "@aws-sdk/client-s3": "1.0.0-ci.0",
-    "@aws-sdk/credential-provider-cognito-identity": "1.0.0-ci.0",
+    "@aws-sdk/client-cognito-identity": "ci",
+    "@aws-sdk/client-s3": "ci",
+    "@aws-sdk/credential-provider-cognito-identity": "ci",
```

Comment:
Are the versions of packages changed on purpose?

Reply:
Yes. Because the local released package version changes all the time. With this change, the dist-tag is preferred over the version number. So you don't need to update it every time after local release.